config/opal_config_asm.m4

```diff
@@ -135,6 +135,7 @@ AC_DEFUN([OPAL_ATOMIC_COMPARE_EXCHANGE_STRONG_TEST_SOURCE],[[
 typedef union {
     uint64_t fake@<:@2@:>@;
     _Atomic __int128 real;
+    __int128 real2;
```

Comment:
I don't see the reason behind this addition. Is this because of compiler warnings or it has a real impact on the detection ?

Reply:
That one is just for warnings. Cleans up some annoying warnings in the config.log.

Reply:
I could also have just changed the type of the variable. Might make that change anyway.